src/query.rs

```diff
@@ -115,8 +176,8 @@ mod tests {
 
     #[test]
     fn all_queries_parse_correctly() {
-        let current_crate = load_pregenerated_rustdoc("template", "new");
-        let indexed_crate = VersionedIndexedCrate::new(&current_crate);
+        let current_crate = &TEST_CRATE_RUSTDOCS["template"].1;
```

Comment:
Wouldn't a `(_previous_crate, current_crate)` be more readable?

Reply:
Instead of `.1`

Reply:
Yeah, good call.

Reply:
Also, `TEST_CRATE_RUSTDOC` kind of suggests that the map contains values that are one crate / one rustdoc, which is not really the case.